sci-libs/opencascade/files/opencascade-7.7.0-musl.patch

```diff
@@ -0,0 +1,250 @@
+diff -upr a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
This patch is pretty big and should really go upstream first.

Reply:
Understood, I'll submit a PR upstream.

Reply:
Thanks!

Reply:
@thesamesam Upstream requires me to sign a CLA with my Mailing Address, which I'm not particularly comfortable giving out. Any alternatives?